src/mdb_v8.c

```diff
@@ -755,7 +775,8 @@ autoconfigure(v8_cfg_t *cfgp)
 		offset_optional = offp->v8o_flags & V8_CONSTANT_OPTIONAL;
 		offset_removed = offp->v8o_flags & V8_CONSTANT_REMOVED;
 		offset_added = offp->v8o_flags & V8_CONSTANT_ADDED;
-		v8_older = v8_version_older(v8_major, v8_minor, cnp->v8c_flags);
+		v8_older = v8_version_older(v8_major,
+		    v8_minor, offp->v8o_flags);
```

Comment:
We were passing the flags of the latest constant that had been processed (`cnp`), but what we really wanted to do is to pass the flags for the current v8_offset being processed.